docs/index.bs

```diff
@@ -1113,6 +1120,12 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
             1. Return {{ClientType/"window"}}.
     </section>
 
+    <section>
+      <h4 id="client-state">{{Client/state}}</h4>
+
+      The <dfn attribute for="Client"><code>state</code></dfn> attribute *must* return the <a>context object</a>'s <a>client state</a>.
```

Comment:
We're not massively consistent here, but we try to use bikeshed shorthand where possible:

```
The <dfn attribute for="Client">state</dfn> attribute *must* return the [=context object=]'s [=Client/client state=].
```